README.md

```diff
@@ -17,4 +17,85 @@ Sample output:
     "percentChange":-90.54688681639148
   }
 }
-```
\ No newline at end of file
+```
```

Comment:
We probably need to add notes section there where we say that you should only use BASECURRENCY_TOKEN notation, like ETH_EOS, not the other way around.

Reply:
@tokenstoredev thought about it a bit: I think it may introduce unnecessary noise to the docs. Currently I simply return a correct error message if wrong order in input. We also always return correct order. Don't think anyone will get an idea to push EOS_ETH after that.

Reply:
Fair enough. Let's make sure to return an error on reversed pairs that we don't support.

Reply:
We still need a notes section that you cannot use reversed pairs in the docs.

Reply:
Fair enough. Let's make sure to return an error on reversed pairs that we don't support.